src/soar_sdk/app.py

```diff
@@ -17,20 +17,58 @@
 from soar_sdk.action_results import ActionOutput
 from soar_sdk.types import Action, action_protocol
 from soar_sdk.logging import getLogger
+import uuid
+
+
+def is_valid_uuid(value: str) -> bool:
+    """Validates if a string is a valid UUID"""
+    try:
+        return str(uuid.UUID(value)).lower() == value.lower()
+    except ValueError:
+        return False
 
 
 class App:
     def __init__(
         self,
         *,
         name: str,
+        app_type: str,
+        logo: str,
+        logo_dark: str,
+        product_vendor: str,
+        product_name: str,
+        publisher: str,
+        appid: str,
+        python_version: str = "3",
```

Comment:
As of 6.4.0, SOAR supports a list here. We should use that format by default in SDK apps
```suggestion
        python_version: list[str] = ["3.9", "3.13"],
```